src/amber/cli/commands/exec.cr

```diff
@@ -37,7 +37,7 @@ module Amber::CLI
           prepare_file
           system("#{options.editor} #{@filename}")
         else
-          File.write(@filename, "puts (#{args.code}).inspect")
+        File.write(@filename, "result = begin\n#{args.code}\nend\nputs result.inspect\n")
```

Comment:
This would probably be more readable if it was broken out into a heredoc:

```
code = <<-CRYSTAL
result = begin
  #{args.code}
end
puts result
CRYSTAL

File.write(@filename, code)
```

Reply:
I would even go a little further and make it its own method

```crytal
def code(args)
  <<-CRYSTAL
  result = begin
    #{args.code}
  end
  puts result
  CRYSTAL
end
```

Reply:
The `code` method doesn't work

Explanation: https://github.com/amberframework/amber/pull/587#issuecomment-361470033